### DISCUSSION TITLE
8.2.5 (Star filed update, bugfix)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#318] [Announcements]
### **Warning! Some older configuration formats not supported anymore! Backup your configuration using `alienfx-config`!**
- In case you upgrade from 7.7.2.1 or early, first install and run [7.8.1](https://github.com/T-Troll/alienfx-tools/releases/tag/7.8.1)
- In case you upgrade from 7.10.5 or early, first install and run [7.10.6.2](https://github.com/T-Troll/alienfx-tools/releases/tag/7.10.6.2)
- In case you upgrade from 8.2.1.2 or early, you will lose all zone light settings in case of rollback, as well as global effect settings.

###  Changes into this release:

`All fan control GUI apps`:
- Bug fixed: Potential bug with incorrect power mode set after resume from sleep in some cases.
- Bug fixed: Power mode can not follow profile settings after profile change.

`AlienFX-GUI`:
- Grid effect changed:
  - If "Keep background" and "Random colors" both active, background color will not be changed
  - "Star field" moved from Trigger to Type and now can be combined with any trigger (except Ambient)
  - "Speed", "Random colors" and "Circle" settings now work for Star Field as well. Gauge type too, but indirect.
- Concurrency access rebuilt for app, decreasing CPU usage and decrease app size about 3%.
- APIv8 global effects now only change then modified and at profile switch.
- Bug fixed: Application resize itself incorrectly after tab switch on some multi-monitor and hi-res monitors.
- Minor fixed and improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/8.2.5'>8.2.5 (Star filed update, bugfix)</a>.</em>

**Comment by XRone99** [comment]
I upgraded from **7.10.6.2** but alienfx-gui won't start
It asks for admin permission (as always) but **crashes** some seconds after the permission
alienfan-gui works correctly in stand alone but i still can't use alienfx-gui

_I followed the instructions in the README file_ (all the stuff about intel drivers etc.) and I can't understand what's wrong

By the way my system is an **Alienware x17 R2** running on Intel i7-12700H, RTX 3080ti Laptop and still works fine whit AlienFX Tools 7.10.6.2

Is there anything I can do to solve this issue in upgrading?

**Reply by T-Troll** [reply]
Most probably it's a configuration issue - check release notes, you can't just update so old version, it requires some intermediate steps!
Hope you backup your old configuration first, as recommended at release notes.

**Reply by XRone99** [reply]
Yes, I followed release notes; it is reported this:

> In case you upgrade from 7.10.5 or early, first install and run [7.10.6.2](https://github.com/T-Troll/alienfx-tools/releases/tag/7.10.6.2)
> In case you upgrade from 8.2.1.2 or early, you will lose all zone light settings in case of rollback, as well as global effect settings.

since I upgrade from 7.10.6.2 I just did a backup and downloaded 8.2.5 and followed the readme file instructions.
Maybe I should first download an intermediate version between 7.10.6.2 an 8.2.5?
Still it seems strange to me

**Reply by T-Troll** [reply]
No, it's correct. But 8.2.5 is quite unstable, so i not wonder it drops. It's just the latest version to support the older zone format...
In fact, you can try to run any version between 8.2.2 and 8.2.5 to convert zone settings.
You can also run latest release, but this can have side effects if zone not converted.
Anyway, it will be useful for diag if you backup current config and share `afx-lights.reg` and `afx-gui.reg`, i'll take a look.

**Reply by XRone99** [reply]
I'll try with 8.2.2, meanwhile this is my current config (running on 7.10.6.2), I don't know were to find _afx-lights.reg_ and _afx-gui.reg_
[Current.csv](https://github.com/T-Troll/alienfx-tools/files/11137941/Current.csv)

**Reply by XRone99** [reply]
I just successfully upgraded from 7.10.6.2 to 8.0.0 but now I can't go over this version
Even upgrading to 8.0.1 causes the program to crash immediately
At the moment I'm still running on 8.0.0 and seems to work fine
But still don't understand why a can't go over this version
if there's anything I can do to help you for diagnostic just ask and i'll try to do it
(I still don't know where to find those .reg files you asked me for)

**Reply by T-Troll** [reply]
Got it. Something wrong with configuration.
Run `alienfx-config backup` (it save 4 .reg files) and send me 2 of them. .csv is your light mappings, keep it!

**Reply by XRone99** [reply]
Ok I did it, GitHub doesn't allow to upload .reg file so I'll put them in a zip for you
[alienfx-tools 8.0.0 backup.zip](https://github.com/T-Troll/alienfx-tools/files/11139209/alienfx-tools.8.0.0.backup.zip)

**Reply by T-Troll** [reply]
Interesting config... Different fan profiles for all!
Ok, let me review and check, then i drop you a message about it.

**Reply by XRone99** [reply]
Thank you

**Reply by T-Troll** [reply]
Looks and works ok for me.
I found one bug, but it's due to the fact i have 2 fans, not 4. ^_^
So.... Right now you can try to run latest release. It should work, but you will lost all zone lights - so you will need to add lights to all zones again. Other zone settings will remain.
Or wait a little, i'll check more.

UPD: This is SPECIAL 8.2.6.2 version with old zone convert support - [alienfx-gui.zip](https://github.com/T-Troll/alienfx-tools/files/11145292/alienfx-gui.zip). All should be OK with it.

**Reply by XRone99** [reply]
8.2.6.2 still crashes right after the start (regardlessly with standard exe and with the special version), I'll keep waiting and in the meantime i'll keep using 8.0.0.
I was wondering maybe I'm doing something wrong...

**Reply by T-Troll** [reply]
Incredible! I check it using your configuration and fake devices, all works well!
Can you please open an issue, and we check seriously, what is providing such effect?

**Reply by XRone99** [reply]
Sorry for delay I've been out of home.
I'm going to open the issue as requested

UPD: Issue opened [ISSUE](https://github.com/T-Troll/alienfx-tools/issues/333)
